install/kubernetes/cilium/values.yaml

```diff
@@ -1,612 +1,1239 @@
+  masterDevice: cni0
 
-    # protocols is the list of protocols to support
-    protocols: tcp,udp
+  # uninstallOnExt enables uninstallation of Cilium BPF programs on flannel
+  # managed pods when the Cilium pod is terminated
+  uninstallOnExit: false
 
-  # nodePort is the configuration for NodePort service handling
-  nodePort:
-    # enabled enables NodePort functionality
-    enabled: false
+# fragmentTracking enables IPv4 fragment tracking support in the datapath.
+# fragmentTracking: true
 
-    # range is the port range to use for NodePort
-    # range:
+# Google Kubernetes Engine
+gke:
+  enabled: false
 
-    # device is the name of the device handling NodePort requests
-    # device:
+# TODO: Add documentation
+healthChecking: true
 
-    # mode is the mode of NodePort feature
-    # mode:
+# TCP port for the agent health API. This is not the port for cilium-health.
+healthPort: 9876
 
-    # algorithm is the name of the load-balancing algorithm for backend selection
-    # e.g. random or maglev
-    # algorithm:
+# hostFirewall enables the enforcement of host policies in the BPF datapath
+hostFirewall: false
 
-    # acceleration is the option to accelerate NodePort via XDP
-    # acceleration:
+# hostPort is the configuration for container hostPort mapping
+hostPort:
+  # enabled enables the hostPort functionality
+  enabled: false
 
-    # bindProtection is the option to enable or disable prevention of
-    # applications binding to service ports
-    bindProtection: true
+# hostServices is the configuration for ClusterIP service handling in host namespace
+hostServices:
+  # enabled enables host reachable functionality
+  enabled: false
 
-    # Append NodePort range to ip_local_reserved_ports if clash with ephemeral
-    # ports is detected
-    autoProtectPortRange: true
+  # protocols is the list of protocols to support
+  protocols: tcp,udp
 
-    # enableHealthCheck enables healthcheck nodePort server for NodePort services
-    enableHealthCheck: true
+hubble:
+  enabled: true
 
-  # hostPort is the configuration for container hostPort mapping
-  hostPort:
-    # enabled enables the hostPort functionality
-    enabled: false
+  # eventQueueSize: ""
 
-  # externalIPs is the configuration for ExternalIPs service handling
-  externalIPs:
-    # enabled enables ExternalIPs functionality
-    enabled: false
+  # flowBufferSize: ""
 
-  # maglev is the configuration for consistent hashing
-  # maglev:
-    # tableSize is the size (parameter M) for the backend table of one service entry
-    # tableSize:
+  # listenAddress: ":4244"
 
-    # hashSeed is the cluster-wide base64 encoded seed for the hashing
-    # hashSeed:
+  metrics:
+    enabled: ""
+    port: 9091
+    serviceMonitor:
+      enabled: false
 
-  # flannel is the flannel specific configuration
-  flannel:
-    # enabled enables the flannel integration
+  metricsServer: ""
+  socketPath: /var/run/cilium/hubble.sock
+  tls:
+    enabled: true
+    auto:
```

Comment:
Documentation strings were also pruned here it seems. Please, add them back :pray: .

Reply:
added